src/App.tsx

```diff
@@ -1,167 +1,46 @@
-import { Loader } from './components/Loader';
-
 import './App.scss';
+import { NavLink, Outlet, useLocation } from 'react-router-dom';
+import classNames from 'classnames';
+import { HomePage } from './components/HomePage';
+
+export const App = () => {
+  const { pathname } = useLocation();
+  const NLClass = ({ isActive }: { isActive: boolean }) => {
+    return classNames('navbar-item', {
+      'has-background-grey-lighter': isActive,
+    });
+  };
```

Comment:
Consider moving the `NLClass` function outside of the `App` component. Defining it inside the component causes it to be recreated on every render, which can affect performance. Instead, define it outside and pass any necessary props to it.